src/js/controllers/customAmount.js

```diff
@@ -69,15 +69,13 @@ angular.module('copayApp.controllers').controller('customAmountController', func
 
   $scope.shareAddress = function() {
     if (!platformInfo.isCordova) return;
-    var protocol = 'bitcoin';
-    if ($scope.wallet.coin == 'bch') protocol += 'cash';
+    var protocol = 'bitcoinz';
```

Comment:
I would argue that a `protocol` variable here isn't necessary anymore (and below).